armi/reactor/converters/uniformMesh.py

```diff
@@ -867,6 +837,64 @@ def _mapStateFromReactorToOther(
         self._cachedReactorCoreParamData = {}
 
 
+class GammaUniformMeshConverter(NeutronicsUniformMeshConverter):
+    """
+    A uniform mesh converter that specifically maps gamma parameters.
+    """
+
+    REACTOR_PARAMS_TO_MAP = {
+        "in": [parameters.Category.neutronics],
+        "out": [parameters.Category.neutronics],
+    }
+    BLOCK_PARAMS_TO_MAP = {
+        "in": [parameters.Category.detailedAxialExpansion],
+        "out": [
+            parameters.Category.detailedAxialExpansion,
+            parameters.Category.multiGroupQuantities,
+            parameters.Category.pinQuantities,
+        ],
+    }
+    # conditions on the output BLOCK_PARAMS_TO_MAP; non-cumulative only
```

Comment:
I would move this comment to the docstring as a `Note`. It's useful for people to see.